include/wx/generic/gridsel.h

```diff
@@ -151,6 +162,16 @@ class WXDLLIMPEXP_CORE wxGridSelection
     wxGrid                              *m_grid;
     wxGrid::wxGridSelectionModes        m_selectionMode;
 
+    // Used by wxGrid::DrawSelectionOverlay() to draw a:
+    //
+    // - Simple rectangle (using wxDC::DrawRectangle() if it is empty and the bounding box is valid.
+    // - Simple polygon (using wxDC::DrawPolygon()) if it represents a simple polygon.
+    // - Poly-polygon (using wxDC::DrawPolyPolygon()) if it consists of multi-polygons.
+    //
+    std::unique_ptr<wxGridPrivate::PolyPolygon> m_polyPolygon;
+
+    size_t                                      m_isAnyLabelHighlighted = 0;
```

Comment:
I'd also appreciate a comment about the meaning of this one as its name (`isXXX`) implies `bool` type for me, but here it is `size_t`, so clearly there is something else going on here, but I don't really know what.